javascript/src/URDFLoader.js

```diff
@@ -271,6 +276,45 @@ class URDFLoader {
             obj.colliders = colliderMap;
             obj.visual = visualMap;
 
+            // Link up mimic joints
+            jointList.forEach(j => {
+
+                if (j.mimicJoint) {
+
+                    if (jointMap.hasOwnProperty(j.mimicJoint)) {
```

Comment:
Considering the full URDF has been parsed here I think it's okay to assume that the parent joint exists so we can remove this check.

Reply:
No problem, done